src/main/java/com/artipie/asto/rx/RxCopy.java

```diff
@@ -66,15 +87,16 @@ public RxCopy(final RxStorage from, final List<Key> keys) {
      * @return The completion signal.
      */
     public Completable copy(final RxStorage to) {
-        return Completable.merge(
-            this.keys.stream()
-                .map(
-                    key -> to.save(
-                        key,
-                        new Content.From(this.from.value(key).flatMapPublisher(cnt -> cnt))
+        return Completable.concat(Flowable.fromIterable(this.keys)
+            .map(
+                key ->
+                    Completable.defer(
```

Comment:
@Sammers21 why to use extra newline here? In other places we're starting expressions and statements on the same line as lambda params